var/spack/repos/builtin/packages/kokkos-tools/package.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class KokkosTools(CMakePackage):
+    """Kokkos Profiling and Debugging Tools"""
+
+    homepage = "https://github.com/kokkos/kokkos-tools/"
+    git = "https://github.com/kokkos/kokkos-tools.git"
+
+    license("Apache-2.0 WITH LLVM-exception")
+
+    version("develop", branch="develop")
+
+    variant("mpi", default=False, description="Enable MPI support")
+    variant("papi", default=False, description="Enable PAPI support")
+
+    depends_on("kokkos")
```

Comment:
Kokkos is NOT a required dependency to build the tools.
You only need it to run the tests/examples.

Reply:
It was easier to just add the kokkos dependency since things like nvtx connector require `Kokkos_ENABLE_CUDA`. You can make it independent and just set that variable via CMake, but that solution seemed more messy and would create duplciation.

Reply:
This dependency on a Kokkos setting for the nvtx and roctx connectors is just silly.  I will make sure it is enabled if CUDA/ROCm are detected.

Reply:
Ping me once it's fixed. I'll update the Spack package.

Reply:
For an initial version, remove the dependence on Kokkos is fine - but we should also make a note somewhere (perhaps in comments or notes in the Spack package) that we don't provide a guarantee of support of vendor connectors, in particular, nvtx-connector/roctx-connector/vtune-connector.

Reply:
I am ok merging as is, and then fixing when we have fixed the build system on the project side.